intellij/org.eclipse.xtext.idea/src/org/eclipse/xtext/idea/build/BuildProgressReporter.xtend

```diff
@@ -57,7 +57,7 @@ class BuildProgressReporter implements BuildRequest.IPostValidationCallback {
 
 	override afterValidate(URI validated, Iterable<Issue> issues) {
 		markAsAffected(validated)
-		for (issue : issues) {
+		for (issue : issues.filterNull) {
```

Comment:
The iterable comes from a language implementation, so we cannot guarantee it. The default impl, shouldn't do that, though.